pkg/api/freight.go

```diff
@@ -19,59 +19,78 @@ import (
 // on its contents and returns it.
 func GenerateFreightID(f *kargoapi.Freight) string {
 	size := len(f.Commits) + len(f.Images) + len(f.Charts)
```

Comment:
This should probably have `len(f.OtherCommits)` as well

Reply:
Ack. Will fix.